### PR TITLE
fixed write permissions for admins and UI bugs. #161874756

### PR DESCRIPTION
**Pull request by grpse**
Signed-off-by: Gilberto Ribeiro <gilbertoribeiropazdarosa@gmail.com>

### Why

Users should change their subscription to anonymous if they want. Admins could change this setting too.

### Wrap up checklist

- [ ] All new code has tests
- [ ] Comments's added to columns / views / functions ([ADD COMMENT](https://www.postgresql.org/docs/9.4/static/sql-comment.html)...) 
- [ ] Code is documented on docs? link PR from [docs repo](https://github.com/common-group/common)
- [ ] Code is reviewed

### CODE REVIEW

File: services/common-api/vendor/gems/common_models/lib/common_models/models/user.rb [hunk 1]
```diff
@@ -13,6 +13,7 @@ class User < ActiveRecord::Base
     has_many :user_api_keys
     belongs_to :platform
     belongs_to :address
+    belongs_to :user_role
```
Comment (devton) [inline services/common-api/vendor/gems/common_models/lib/common_models/models/user.rb]: user has_many user_roles

File: services/common-api/vendor/gems/common_models/lib/common_models/models/user_role.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# coding: utf-8
+module CommonModels
+    class UserRole < ActiveRecord::Base
+      self.table_name = 'community_service.user_roles'
+      belongs_to :platform
+      has_many :user
```
Comment (devton) [inline services/common-api/vendor/gems/common_models/lib/common_models/models/user_role.rb]: user_role belongs to user

File: services/common-api/app/controllers/v1/api_base_controller.rb [hunk 3]
```diff
@@ -43,6 +43,11 @@ def is_platform_user?
       jwt_role == 'platform_user'
     end
 
+    def is_admin?
+      user_role = current_platform.user_roles.find_by_user_id jwt_user_id
+      user_role.role_name == 'admin'
```
[the remark this answers is not in the document]
Comment (grpse) [reply services/common-api/app/controllers/v1/api_base_controller.rb]: I used instead:
    def is_admin?
      user_role = current_platform.user_roles.find_by_user_id jwt_user_id
      role_name = user_role[:role_name].presence || ''
      role_name == 'admin'
    end

File: services/common-api/vendor/gems/common_models/lib/common_models/models/user.rb (hunk 1 above)
Comment (grpse) [reply services/common-api/vendor/gems/common_models/lib/common_models/models/user.rb]: done

File: services/common-api/vendor/gems/common_models/lib/common_models/models/user_role.rb (hunk 2 above)
Comment (grpse) [reply services/common-api/vendor/gems/common_models/lib/common_models/models/user_role.rb]: done

File: services/common-api/app/controllers/v1/api_base_controller.rb [hunk 4]
```diff
@@ -45,7 +45,8 @@ def is_platform_user?
 
     def is_admin?
       user_role = current_platform.user_roles.find_by_user_id jwt_user_id
-      user_role.role_name == 'admin'
+      role_name = user_role[:role_name].presence || ''
```
Comment (devton) [inline services/common-api/app/controllers/v1/api_base_controller.rb]: if user_role is nil the [:role_name] will break.

use:
role_name = user_role.try(:[], :role_name)

Comment (grpse) [reply services/common-api/app/controllers/v1/api_base_controller.rb]: I replaced the code.

**Pull request merged**